bridge/engine/EngineStreamDirector.h

```diff
@@ -547,9 +635,28 @@ class EngineStreamDirector
         return false;
     }
 
+    void setSlidesSsrcAndBandwidth(size_t slidesSsrc, uint32_t bwKbps)
+    {
+        _slidesSsrc = slidesSsrc;
+        _slidesBandwidthKbps = bwKbps;
+    }
+
 private:
-    static const size_t lowQuality = 0;
-    static const size_t midQuality = 1;
+    static constexpr size_t lowQuality = 0;
+    static constexpr size_t midQuality = 1;
+    static constexpr size_t highQuality = 2;
+    static constexpr size_t dropQuality = 3;
+
+    enum class ConfigLadderCols
```

Comment:
I think this should be a struct instead. Make the configLadder an array of such structs with ascending or descending quality. Make an aneum of the QualityLevels above.
If you would have performed actual matrix operations I would have suggested the math::Matrix, but it seems we do not.

Reply:
an put a comment above the struct that all bandwidths are in kbps.

Reply:
Fixed.